src/cata_tiles.cpp

```diff
@@ -1125,7 +1125,7 @@ bool cata_tiles::draw_from_id_string(std::string id, TILE_CATEGORY category,
     }
 
     if (it == tile_ids.end()) {
-        long sym = -1;
+        wchar_t sym = -1;
```

Comment:
In other words, if you want to make this unsigned, the method needs to be refactored to not try and assign a negative number to it.